src/sider/senoppfolging/OvingssideLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Heading, HelpText } from "@navikt/ds-react";
+import React from "react";
+import { EksternLenke } from "@/components/EksternLenke";
+
+const texts = {
+  heading: "Øvingsside for Snart slutt på sykepengene",
```

Comment:
Det er kanskje litt kryptisk med en øvingsside for "Snart slutt på sykepengene" 🤔 Hva med å bare skrive direkte at de kan gå inn å se hvordan skjemaet som blir sendt fra sykmeldt ser ut? 😊
```suggestion
  heading: "Se hvordan skjema ser ut for sykmeldt",
```

Reply:
Tanken var å gjøre det tydelig at det er en øvingsside eller demo av skjemaet slik at veilederne ikke er redd for å trykke seg igjennom skjemaet. Ble kommentert på forrige møte med veilederne i pilot at de var litt redde for å trykke seg igjennom skjemaet. Samtidig representerer demoen bare den nyeste versjonen av skjemaet og ikke nødvendigvis hvordan skjemaet så ut for den sykmeldte når vedkommende svarte på skjemaet. Men når du sier det så er jeg enig i at overskriften høres litt kryptisk ut, jeg kan høre hva Peter tenker om det. 😊

Reply:
Peter foreslo å bytte tittel til "Sykmeldtes svarskjema"

Reply:
Fin tittel det 😊 Da er det bare viktig å poengtere at det er en demoside og ikke det faktiske svarskjemaet til den faktiske sykmeldte

Reply:
Ja, hvordan tenker du da? Kunne det stått i linken? 
`link: "Se nyeste versjon av svarskjemaet her (demo)"`
Eller tenkte du i tittelen?